### PR TITLE
Edit console options

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: src/components/Options/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <DetailContainer>
+      Options
+    </DetailContainer>
+  )
+}
+Options.propTypes = {
+
+}
+
+export default connect(
```
Comment (matobet) [inline src/components/Options/index.js]: `export default connect()(Options)` is equivalent

File: src/components/VmDetail/style.css [hunk 2]
```diff
@@ -45,3 +45,17 @@ dd {
...
+}
+*/
+.left-delimiter {
+    padding: 0 13px 0 0;
+}
+
+.left-delimiter + .left-delimiter {
+    padding-left: 13px;
+    border-left: 1px solid #d1d1d1;
+}
```
Comment (matobet) [inline src/components/VmDetail/style.css]: please have your IDE put newlines at the end of all files automatically :)

File: src/reducers/vms.js [hunk 3]
```diff
@@ -91,6 +91,8 @@ function vms (state, action) {
       return state.delete('selected')
     case 'VM_ACTION_IN_PROGRESS':
       return state.setIn(['vms', action.payload.vmId, 'actionInProgress', action.payload.name], action.payload.started)
+    case 'SET_VM_CONSOLES':
+      return state.setIn(['vms', action.payload.vmId, 'consoles'], action.payload.consoles) // JS object, not immutable
```
Comment (matobet) [inline src/reducers/vms.js]: are you really sure it is good to mix mutable and immutable objects like this? I mean if it is part of the store and it is not just the top-level namespacing object `{ vms, config, ... }` for reducers I think we should rather go with full immutable...

**Review by matobet — CHANGES_REQUESTED** [review]

File: src/components/VmDetail/style.css (hunk 2 above)
Comment (mareklibra) [reply src/components/VmDetail/style.css]: Done

File: src/components/Options/index.js (hunk 1 above)
Comment (mareklibra) [reply src/components/Options/index.js]: Done

**Review by matobet — APPROVED** [review]

**Pull request merged**